flake.nix

```diff
@@ -0,0 +1,31 @@
+{
+  description = "Hardware and Firmware of the Solo Quadruped Master Board.";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
+  };
+
+  outputs =
+    { self, nixpkgs }:
+    let
+      pkgs = nixpkgs.legacyPackages.x86_64-linux;
+    in
+    {
+      packages.x86_64-linux.sdk = pkgs.stdenv.mkDerivation {
+
+        pname = "odri-masterboard-sdk";
+        version = "nightly";
+
+        src = builtins.path {
+          name = "sdk";
+          path = ./sdk/master_board_sdk;
+        };
+
+        nativeBuildInputs = with pkgs; [
+          cmake
+          python312
+          python312Packages.boost
```

Comment:
`ldd result/lib/python3.12/site-packages/libmaster_board_sdk_pywrap.cpython-312-x86_64-linux-gnu.so` has `libboost_python312.so.1.87.0 => /nix/store/snwnv20gazn0zp2gvg86j1hrcq5ynmgj-boost-1.87.0/lib/libboost_python312.so.1.87.0`, so I think this dependency should be in `propagatedBuildInputs`